puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/builders/systest.yaml

```diff
@@ -46,7 +46,8 @@
               variables:
                 bats_environment:
                 - FOREMAN_EXPECTED_VERSION: ${{expected_version}}
-                ${{args:+foreman_installer_options:${{args}}}}
+                foreman_installer_options:
+                - '${{args}}'
```

Comment:
should this be intented over one?

```yaml
foreman_installer_options:
  - '${{args}}'
```

Reply:
Perhaps, I wanted to keep it consistent with line 48, so I'll indent that as well.